docs/setup/aks/getting-started-aks.md

```diff
@@ -0,0 +1,149 @@
+```
+
+## Granting Kubernetes access to ACR
+1. To use AKS, you'll need to set up a secret and mount it to the playground container, so that it can authenticate its Docker client against ACR. Start by creating a local directory on your machine, and inside create a new file for storing the credentials:
+```
+File name: <acr-user-name>---<acr-server>.json
+File content: <acr-password>
+```
+
+2. The expected format of the secret is a text file containing the password of the user specified by the file name. For example, if the secret name is `966e5820-5443-48c5-be62-b4d62798ab68---mynuclioacr.azurecr.io.json` and the contents are `aaabbbcccddd`, the playground will run "docker login" where:
+```
+- user: 966e5820-5443-48c5-be62-b4d62798ab68
+- password: aaabbbcccddd
+- server: mynuclioacr.azurecr.io
+```
+
+3. Use the CLI to navigate to the directory where you stored the secrets file. Create the Kubernetes secret from the service-key file, and delete the file:
+```sh
+kubectl create secret generic nuclio-docker-keys --from-file=966e5820-5443-48c5-be62-b4d62798ab68---mynuclioacr.azurecr.io.json
+
+rm 966e5820-5443-48c5-be62-b4d62798ab68---mynuclioacr.azurecr.io.json
+```
+
+4. Create a `configmap` file that will be used by the playground to determine which repository should be used for pushing and pulling images:
+```sh
+kubectl create configmap nuclio-registry --from-literal=registry_url=mynuclioacr.azurecr.io
+```
+> **Note**: While the secret is used to trigger a docker login to a docker registry, the config map is used to pass a variable indicating which repository to push to. 
+
+## Install Nuclio
+By now you should have a functioning Kubernetes cluster, a Docker registry, and a working Kubernetes CLI (kubectl). Now, you can go ahead and install the nuclio services on the cluster.
+
+1. Deploy the `nuclio controller`, which watches for new functions, by running the following kubectl command:
+```sh
+kubectl apply -f https://raw.githubusercontent.com/nuclio/nuclio/master/hack/k8s/resources/controller.yaml
+```
+
+2. Now, you can deploy the `nuclio playground` and access it on port 8070 of a relevant node IP:
+```sh
+kubectl apply -f https://raw.githubusercontent.com/nuclio/nuclio/development/hack/aks/resources/playground.yaml
+```
+
+3. Add `Traefik Load Balancer` to act as an Ingress Controller for your Nuclio Functions:
+```sh
+kubectl apply -f https://raw.githubusercontent.com/nuclio/nuclio/development/hack/aks/resources/traefik-lb.yaml
+```
+
+Finaly, let’s wait until the services will be up and running.
+Use the command `kubectl get pods` to verify that both the controller and playground have a status of `Running`. For more information about `kubectl`, see the [Kubernetes documentation](https://kubernetes.io/docs/user-guide/kubectl-overview/).
+
+## Deploy a function with the nuclio playground
+Find your nuclio-playgound ExternalIP by typing: `kubectl get services -o wide`.
```

Comment:
I think you should also add port-forward in case you don't want to expose Playground with LoadBalancer

`kubectl port-forward $(kubectl get pod -l app=nuclio-playground -o jsonpath='{.items[0].metadata.name}') 8070:8070 &`